_posts/spells/fourth-level/2020-10-15-spells-4-confusion.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Confusion
+permalink: "/spells/fourth-level/confusion/"
+layout: spell-post
+categories: [Spells, Fourth Level]
+tags: [Bard,Druid,Sorcerer,Wizard]
+source: "Basic Rules (PHB)"
+spell-level: "4th-level"
+school: Enchantment
+casting-time: "1 Action"
+range: "60 feet"
+components: "V,S,M (three nut shells)"
+duration: "Concentration, up to 1 minute"
+---
+
+This spell assaults and twists creatures' minds, spawning delusions and provoking uncontrolled action. Each creature in a 10-foot-radius sphere centered on a point you choose within range must succeed on a Wisdom saving throw when you cast this spell or be affected by it.
+
+An affected target can't take reactions and must roll a d10 at the start of each of its turns to determine its behavior for that turn.
+
+***Confusion Behavior (Roll D10)***
```

Comment:
Put this info into a table, please! 

An example table might look like in markdown:
|d10| behavior |
|:---:|:-----------|
|1|adsfoajdsfojfdsa|

Reply:
can do!